TLM/TLM/TrafficLight/Impl/CustomSegmentLights.cs

```diff
@@ -190,68 +222,84 @@ protected CustomSegmentLights(ICustomSegmentLightsManager lightsManager, ushort
 
         public bool IsAnyRightGreen() {
```

Comment:
Missing line feed before `return` in unchanged `IsAnyMainGreen()` method above.

Reply:
Also inconsistent brackets in that same method.

Reply:
Fixed lines and curly braces in that method